src/components/sections/ProductGallery/ProductGalleryPage.tsx

```diff
@@ -49,7 +85,30 @@ function GalleryPage({ page, display, title, fallbackData }: Props) {
         pageSize={itemsPerPage}
         title={title}
       />
-      <ProductGrid products={products} page={page} pageSize={itemsPerPage} />
+      {/* If showSponsoredProducts is on, a ProductTiles will be display in between two blocks of ProductGrid on the page 0 */}
```

Comment:
It's better to move this comment to the props type definition.

Reply:
Well pointed!